src/r5py/r5/travel_time_matrix_computer.py

```diff
@@ -45,6 +45,14 @@
 }
 
 
+class NoIDColumnError(Exception):
```

Comment:
Not sure but should we perhaps have a separate file for these errors? That's what I've typically done. Thoughts, @christophfink?

Reply:
Happy to do that - suggestions on naming convention/location?

`util/errors.py` perhaps?

Reply:
sounds great! 

Can we: 
- call the file `util/exceptions.py` (I’d expect more to come), and 
- inherit from 
    - a more specific exception (maybe `ValueError`?), **and** 
    - a generic R5pyException (so we have the possibility to catch all r5py errors with `except R5pyException:` )